### PR TITLE
GTX-512 - Implemented a recover mechanism to try and automatically ge…

### PR DESCRIPTION
**Pull request by cma06**
…t past free spin bonus rounds

### CODE REVIEW

File: skyVegasTests/stakeUpdateGameClient.spec.ts
Comment (Wilko2001) [inline skyVegasTests/stakeUpdateGameClient.spec.ts:1]: Is there a reason the first "await page.keyboard.press('Enter', { delay: 500 });" was removed? I thought we needed that in to ensure we got onto the game properly?

Comment (cma06) [reply skyVegasTests/stakeUpdateGameClient.spec.ts:1]: We moved the logic which gets past the first game menu to the launchGame function so this was not needed anymore

Comment (Wilko2001) [reply skyVegasTests/stakeUpdateGameClient.spec.ts:1]: Ah sorry I can see this now, my bad. Thanks for clarifying!

**Review by Wilko2001 — APPROVED** [review]

File: skyVegasTests/stakeUpdateGameClient.spec.ts
Comment (cma06) [reply skyVegasTests/stakeUpdateGameClient.spec.ts:1]: No worries, Good question

**Pull request merged**